test/scrabble_test.rb

```diff
@@ -4,9 +4,48 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  def test_it_exists
+    game = Scrabble.new
+    assert_instance_of Scrabble, game
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    game = Scrabble.new
+    assert_equal 1, game.score("a")
```

Comment:
I would have kept a test for scoring a letter with a greater point value than 1 - just to make sure that worked as well.